utils/cliutils/commandsflags.go

```diff
@@ -1360,7 +1361,7 @@ var commandFlags = map[string][]string{
 		global, serverIdResolve, serverIdDeploy, repoResolve, repoDeploy,
 	},
 	GoPublish: {
-		deps, self, url, user, password, apikey, accessToken, serverId, buildName, buildNumber, module, project,
+		deps, url, user, password, apikey, accessToken, buildName, buildNumber, module, project,
```

Comment:
I may be wrong here - but doesn't this mean that the people who use the **--self** flag will break, because this flag iswas removed?